.changeset/sour-jobs-count.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/container-utils": minor
+---
+
+Move DeltaManagerProxyBase from container-utils to container-runtime
+
+Moves the `DeltaManagerProxyBase` from `@fluidframework/container-utils` into `@fluidframework/container-runtime`, which is the only place that currently uses it. It is no longer exported, and no replacement API is intended.
```

Comment:
Was this deprecated already? If not, can we deprecate it in 6.1?

Reply:
Not yet, no.  None of our partners are using it, but I suppose it would still be good form to do that. I'll put up a separate PR doing that in main.

Reply:
PR deprecating the API on main: #16813

Reply:
I'll update the details in this PR once the other PR has gone through and been merged into next.

Reply:
PR merged into main. Will wait until it has been merged into next before updating this PR.

Reply:
I messed up a merge. I'm just going to open a fresh PR rather than trying to clean up the history.

Reply:
Replacement PR: #16875